pkg/integrations/postgres_exporter/postgres_exporter_test.go

```diff
@@ -0,0 +1,24 @@
+package postgres_exporter
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_ParsePostgresURL(t *testing.T) {
+	dsn := "postgresql://linus:42secret@localhost:5432/postgres?sslmode=disable"
+	expected := map[string]string{
+		"dbname":   "'postgres'",
+		"host":     "'localhost'",
+		"password": "'42secret'",
+		"port":     "'5432'",
+		"sslmode":  "'disable'",
+		"user":     "'linus'",
```

Comment:
We should make sure that the quotes postgres add also get trimmed out since we use these values for deriving an instance label:

```suggestion
		"dbname":   "postgres",
		"host":     "localhost",
		"password": "42secret",
		"port":     "5432",
		"sslmode":  "disable",
		"user":     "linus",
```